runtime/compiler/p/codegen/J9UnresolvedDataSnippet.cpp

```diff
@@ -66,6 +67,13 @@ J9::Power::UnresolvedDataSnippet::UnresolvedDataSnippet(
    {
    }
 
+uint8_t *J9::Power::UnresolvedDataSnippet::getAddressOfDataReference()
+   {
+   if (self()->getDataReferenceInstruction())
```

Comment:
Will this return null at the time of the merge?

Reply:
Yes, this currently returns `NULL` on Power. Other codegens do use this, but the Power codegen doesn't ever populate this field.